src/test/java/org/mockitousage/verification/BasicVerificationTest.java

```diff
@@ -47,8 +47,11 @@ public void shouldFailVerificationOnMethodArgument() throws Exception {
         verify(mock).clear();
         try {
             verify(mock).add("bar");
-            fail();
-        } catch (AssertionError expected) {}
+        } catch (AssertionError expected) {
```

Comment:
Empty catch again? :) AssertJ or suppress will work better for us I think.

Reply:
done. Will look nicer once we can use lambdas